<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacientEdit</class>
 <widget class="QWizardPage" name="PacientEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editácia pacienta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/icons/user-properties.png</normaloff>:/icons/user-properties.png</iconset>
  </property>
  <property name="title">
   <string>Údaje o pacientovi</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="rodneCisloLabel">
     <property name="text">
      <string>&amp;Rodné číslo</string>
     </property>
     <property name="buddy">
      <cstring>rodneCisloEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="MenoRCEdit" name="rodneCisloEdit">
     <property name="inputMask">
      <string>999999/9999; </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="menoLabel">
     <property name="text">
      <string>&amp;Meno a priezvisko</string>
     </property>
     <property name="buddy">
      <cstring>menoEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MenoRCEdit" name="menoEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="datumLabel">
     <property name="text">
      <string>&amp;Dátum narodenia</string>
     </property>
     <property name="buddy">
      <cstring>datumNarodeniaEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="datumNarodeniaEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="cisloPoistovneLabel">
     <property name="text">
      <string>Čí&amp;slo poisťovne</string>
     </property>
     <property name="buddy">
      <cstring>cisloPoistovneEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="cisloPoistovneEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="adresaLabel">
     <property name="text">
      <string>&amp;Adresa</string>
     </property>
     <property name="buddy">
      <cstring>adresaEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="adresaEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="zoznamAlergiiLabel">
     <property name="text">
      <string>&amp;Zoznam alergií</string>
     </property>
     <property name="buddy">
      <cstring>zoznamAlergiiList</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="odoberAlergiuButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="data.qrc">
         <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="pridajAlergiuButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="data.qrc">
         <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QListView" name="zoznamAlergiiList"/>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="errorMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenoRCEdit</class>
   <extends>QLineEdit</extends>
   <header>MenoRCEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rodneCisloEdit</tabstop>
  <tabstop>menoEdit</tabstop>
  <tabstop>datumNarodeniaEdit</tabstop>
  <tabstop>cisloPoistovneEdit</tabstop>
  <tabstop>adresaEdit</tabstop>
  <tabstop>zoznamAlergiiList</tabstop>
  <tabstop>odoberAlergiuButton</tabstop>
  <tabstop>pridajAlergiuButton</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
